packages/odyssey-react/src/components/Checkbox/Checkbox.module.scss

```
@@ -155,8 +155,8 @@
   .box {
     width: var(--BoxSize);
     height: var(--BoxSize);
-    border: var(--BorderSize) var(--BorderStyle) var(--ColorBorder);
-    background-color: var(--ColorBackground);
+    border: var(--BorderSize) var(--BorderStyle) var(--BorderColor);
```

```suggestion
    border-width: var(--BorderWidth)
    border-style: var(--BorderStyle)
    border-color: var(--BorderColor);
```
would be nice to break apart the short hand here to help with readability! Also `BorderWidth` not `BorderSize` !